### PR TITLE
Fix bug where no listed venue crashed app

### PR DESCRIPTION
**Pull request by nicole-a-tesla**
Surround json parsing logic with try/except blocks and provide
defaults for missing keys.

### CODE REVIEW

**Comment by nicole-a-tesla** [comment]
@ecmendenhall What are some options for refactoring our changes to the meetup_api_response_parser module?

**Comment by ecmendenhall** [comment]
@nicole-a-tesla Nothing is something. :smile_cat:

File: meetup/services/meetup_api_response_parser.py [hunk 1]
```diff
@@ -3,9 +3,27 @@ def parse(api_response):
...
 
   if json:
-    parsed_response['venue'] = json[0].get('venue').get('name')
-    parsed_response['next_event_topic'] = json[0].get('name')
-    parsed_response['datetime'] = json[0].get('time')
-    parsed_response['lat'] = json[0].get('group').get('lat')
-    parsed_response['lon'] = json[0].get('group').get('lon')
+
+    try:
+      parsed_response['venue'] = json[0].get('venue').get('name')
```
Comment (ecmendenhall) [inline meetup/services/meetup_api_response_parser.py:8]:
``` ipy
In [1]: {}.get('huh')

In [2]: {}.get('huh').get('wut')
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-80cb2343ebe2> in <module>()
----> 1 {}.get('huh').get('wut')

AttributeError: 'NoneType' object has no attribute 'get'

In [3]: {}.get('huh', {}).get('wut')

In [4]:
```

Comment (ecmendenhall) [inline meetup/services/meetup_api_response_parser.py:8]: ...but a better refactoring is called the Null Object pattern: http://refactoring.com/catalog/introduceNullObject.html

File: meetup/tests.py [hunk 2]
```diff
@@ -219,5 +220,5 @@ def test_parsed_response_contains_event_datetime(self):
     self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('datetime'), 1458730800000)
 
   def test_parsed_response_contains_lat_and_long(self):
-    self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('lat'), 40.7599983215332)
-    self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('lon'), -73.98999786376953)
+    self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('lat'), 40.75501251220703)
```
Comment (ecmendenhall) [inline meetup/tests.py:223]: I don't understand why this changed. Is it possible your tests are talking to a real API?

**Pull request merged**